nyaa/templates/rss.xml

```diff
@@ -40,6 +40,11 @@
 			<nyaa:categoryId>{{- cat_id }}</nyaa:categoryId>
 			<nyaa:category>  {{- category_name(cat_id) }}</nyaa:category>
 			<nyaa:size>      {{- torrent.filesize | filesizeformat(True) }}</nyaa:size>
+			<nyaa:submitter> {{- 'Anonymous' if torrent.anonymous or not torrent.user else torrent.user.username }}</nyaa:submitter>
+			<nyaa:trusted>	 {{- torrent.trusted }}</nyaa:trusted>
+			<nyaa:remake>	 {{- torrent.remake }}</nyaa:remake>
+			<nyaa:summary><![CDATA[{{- torrent.description or '#### No description.' }}]]></nyaa:summary>
```

Comment:
Not available on [ES results](https://github.com/nyaadevs/nyaa/blob/master/es_mapping.yml#L84).

Reply:
How can I make available on Elastic Search?

Reply:
You can't. We don't store the description in ES, and it's not really worth the overhead to add it just for RSS.

Reply:
Should I make descriptions only available in none ES queries?

Reply:
Nah. It breaks consistency, pure-db queries are rate for RSS, few would have any use for the field (it's not visible for human-RSS readers), plus it's unprocessed markdown.

Reply:
You're right people don't really need the summary since they can use the other info from the feed. I'll remove it then since I don't need it either.

What should I do about your other change request? The one for the submitter field?